modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+Cluster administrators can use the OpenTelemetry Protocol (OTLP) output to collect and forward logs to OTLP receivers. The OTLP output uses the specification defined by the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability framework] to send data over HTTP with JSON encoding.
+
+:FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* Create or edit a `ClusterLogForwarder` custom resource (CR) to enable forwarding using OTLP by adding the following annotation:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled" # <1>
+  name: clf-otlp
+spec:
+  serviceAccount:
+    name: <service_account_name> 
+  outputs:
+  - name: otlp
+    type: otlp
+    otlp:
+      tuning:
+        compression: gzip 
+        deliveryMode: AtLeastOnce
+        maxRetryDuration: 20
+        maxWrite: 10M
+        minRetryDuration: 5
+      url: <otlp_url> # <2>
+  pipelines:
+  - inputRefs:
+    - application
+    - infrastructure
+    - audit
+    name: otlp-logs
+    outputRefs:
+    - otlp
+----
+<1> Use this annotation to enable the OpenTelemetry Protocol (OTLP) output, which is a Technology Preview feature.
```

Comment:
Not wrong to indicate TP status here, but you already have the TP notice above, so is this really necessary? It might not get changed when the feature transitions to GA. Your call though.

Reply:
Eh, this file isn't my commit, so I'll leave that up to the author. Good eyes, though!